app/scripts/app-init.js

```diff
@@ -184,12 +184,33 @@ const registerInPageContentScript = async () => {
   }
 };
 
-chrome.runtime.onInstalled.addListener(function (details) {
+/**
+ * `onInstalled` event handler.
+ *
+ * On MV3 builds we must listen for this event in `app-init`, otherwise we found that the listener
+ * is never called.
+ * For MV2 builds, the listener is added in `background.js` instead.
+ *
+ * @param {object} details - Event details.
+ * @param {string} details.reason - The reason that this event was dispatched.
```

Comment:
very non blocking nit: we have types for (most of) the extension apis. `chrome.runtime.InstalledDetails` should do the trick in this case.